paywall/src/pages/about.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import LandingPage from '../components/LandingPage'
+
+export default function NextLandingPage() {
```

Comment:
Does this need to be in this format vs the `const Blah = () => {` ES6 construction we've used elsewhere?

Also, `NextLandingPage` again seems like a superfluous description - it's a page and it uses next. Could we just call it `Landing`?